src/lwc/utilAlertBanner/utilAlertBanner.html

```diff
@@ -1,13 +1,31 @@
 <template>
-    <lightning-layout>
-        <lightning-layout-item size="12">
-            <div class={alertTheme} role="alert">
-                <span class="slds-assistive-text">{alertAssistText}</span>
-                <span class="slds-icon_container slds-icon-utility-error slds-m-right_x-small">
-                    <lightning-icon icon-name={alertIcon} variant={alertVariant} size={alertSize}></lightning-icon>
-                </span>
-                <h2>{alertMessage}</h2>
-            </div>
-        </lightning-layout-item>
-    </lightning-layout>
+    <template if:true={isVisible}>
```

Comment:
This is just a UI discrepancy I noticed, in other sections of NPSP Settings, we display UI messages using this component: 
![image](https://user-images.githubusercontent.com/20406258/102513735-d0cf4680-4048-11eb-964e-5832ef5ee11e.png)

We seem to be using a different component for this WI: 
![image](https://user-images.githubusercontent.com/20406258/102513830-f2c8c900-4048-11eb-8e1a-e9ee4c88518f.png)

Maybe we should align with the existing UI ? What are your thoughts?

Reply:
Discussed in our call today. We are going to leave the style difference as is since we are using LWC here and the top alert is using a Visualforce component.